packages/bot-web-ui/src/components/load-modal.jsx

```diff
@@ -0,0 +1,310 @@
+        <Icon icon={'IcMinusRounded'} className='recent__preview-icon' onClick={() => onZoomInOutClick(false)} />
+    </div>
+);
+
+const Recent = ({
+    explaination_expand,
+    getRecentFileIcon,
+    loadFileFromRecent,
+    onExplainationToggle,
+    previewWorkspace,
+    recent_files,
+    selected_file,
+    ...props
+}) => (
+    <div className='recent__container'>
+        {
+            recent_files.length ?
+                <>
+                    <div className='recent__content'>
+                        <div className='recent__files load__content-with-footer'>
+                            <div className='recent__title'>{localize('Recent')}</div>
+                            <div className='recent__list'>
+                                {
+                                    recent_files.map(file => {
+                                        return (
+                                            <div
+                                                className={classnames(
+                                                    'recent__item',
+                                                    { 'recent__item--selected': selected_file === file.id },
+                                                )}
+                                                key={file.id}
+                                                onClick={() => previewWorkspace(file)}
+                                            >
+                                                <div className='recent__item-text'>
+                                                    <span className='recent__item-title'>{file.name}</span>
+                                                    <span className='recent__item-time'>{timeSince(file.timestamp)}</span>
+                                                </div>
+                                                <div className='recent__item-location'>
+                                                    <Icon
+                                                        icon={getRecentFileIcon(file.location)}
+                                                        className={classnames({ 'gd__icon--active': file.location === 'google drive' })}
+                                                    />
+                                                    <div className='recent__item-saved'>{file.location.replace(/\b\w/g, l => l.toUpperCase())}</div>
+                                                </div>
+                                            </div>
+                                        );
+                                    })
+                                }
+                            </div>
+                        </div>
+                        <div className='recent__preview load__content-with-footer' >
+                            <div className='recent__title'>{localize('Preview')}</div>
+                            <div id='scratch_recent' className='recent__preview-workspace' >
```

Comment:
I think it's better to stick to the `recent_` prefix, even for ids e.g. `recent__scratch-workspace`.